src/main/java/org/dswarm/graph/parse/BaseNeo4jHandler.java

```diff
@@ -306,10 +317,19 @@ public Relationship deprecateStatement(final String uuid) throws DMPGraphExcepti
 
 		try {
 
-			final Relationship rel = getRelationship(uuid);
+			final Optional<Relationship> optionalRel = processor.getRelationshipFromStatementIndex(uuid);
+
+			if(!optionalRel.isPresent()) {
+
+				BaseNeo4jHandler.LOG.error("couldn't find statement with the uuid '{}' in the database", uuid);
+			}
+
+			final Relationship rel = optionalRel.get();
```

Comment:
if it is not present, this will throw (probably a NSEE). Is this desired behavior?